### ISSUE TITLE
DIA-NN undetected protein

### ISSUE THREAD

**Issue by Yukii0102** [vdemichev/DiaNN#911]
Hi.
I am using the software DIA-NN to perform proteomic analysis. 
I am checking protein expression using ".pg files", but the protein which I want to look up is not included in the file.
Could you tell me the solution.
Do you think if I try to run DIA-NN, the protein which I want to look up is detected?

**Comment by vdemichev** [comment]
Means it's just not detectable or does not pass the stringent FDR filter used for the pg_matrix. I would recommend to also check in the main report.

**Comment by Yukii0102** [comment]
Thank you.
1 Is there any possibility to run DIA-NN again to obtain the desired protein?
2 if the protein which I want to look up is not included in DIA-NN "pg file", is there any alternative way to obtain it? I can only think of a method to calculate the peak area using Skyline, Peakview, etc. and give it as a relative ratio of the protein, so please let me know if there is another method.

**Comment by vdemichev** [comment]
No, .tsv. Please see https://github.com/vdemichev/DiaNN?tab=readme-ov-file#output

**Comment by Yukii0102** [comment]
I see.
Thank you very much.
